widget/entry.go

```diff
@@ -131,6 +220,14 @@ type Entry struct {
 	OnCursorChanged         func() `json:"-"`
 
 	focused bool
+
+	// Selection start/end are private as they represent where the selection started, which
```

Comment:
This comment about start/end seems out of date - neither are referenced below

Reply:
Good spotting. I'll fix this comment when I address the comment below.